prelude/prelude.purs

```diff
@@ -120,6 +120,9 @@ module Prelude
   class Functor f where
     (<$>) :: forall a b. (a -> b) -> f a -> f b
 
+  fmap :: forall f a b. (Functor f) => (a -> b) -> f a -> f b
+  fmap = (<$>)
```

Comment:
Well, we can break lots of things before 1.0, but still, let's put it to a vote. Generally, I'd be in favor of using named things by default.